tests/ci_build/Dockerfile.cudf

```diff
@@ -23,7 +23,10 @@ RUN \
 # Install other Python packages
 RUN \
     source activate cudf_test && \
-    pip install numpy pytest scipy scikit-learn pandas matplotlib wheel kubernetes urllib3 graphviz
+    pip install numpy pytest scipy scikit-learn pandas matplotlib wheel kubernetes urllib3 graphviz && \
+    wget https://github.com/rapidsai/dask-cuda/archive/v0.10.0.tar.gz -O ./dask-cuda-v0.10.0.tar.gz && \
```

Comment:
Can we use Conda to install dask-cuda? We can add `dask-cuda` to line 21.

Reply:
@hcho3 Let me try.  docker + pip + conda combined to manage the dependency ...  Sometimes I think there must be something went terribly wrong.

Reply:
@hcho3 Done.